schema/proto3/port.proto

```diff
@@ -26,6 +26,11 @@ import "common.proto";
 message PortConfiguration {
     int32 version = 1;
 
+    // FIXME: the options for NetworkType are:
+    // 1. put in PortConfiguration here since only useful for Port, ACA programming will be direct and easier
+    // 2. put in VPC configuration since it is a "network level" configuration, 
+    //      that mean every port configuration needs to go with VPC config for ACA to look up
+    NetworkType network_type = 13;  // TODO: the ID will be update after discussion
```

Comment:
New field to be placed at the end after discussion.

Reply:
discussion point: since this is a major revision of the schema, I am proposing to make the fields pretty with the ordering and not to add new fields at the end. Compatibility with the previous release of Alcor or ACA was not a goal in my mind.

Reply:
+1